utility/src/main/java/org/oppia/android/util/logging/firebase/FirebaseEventLogger.kt

```diff
@@ -57,8 +66,20 @@ class FirebaseEventLogger private constructor(
      * Returns a new [FirebaseEventLogger] for the current application context.
      *
      * Generally, only one of these needs to be created per application.
+     *
+     * This [FirebaseEventLogger] implements the [EventLogger] for facilitating analytics log
+     * reporting.
      */
     fun create(): EventLogger =
       FirebaseEventLogger(firebaseAnalytics, networkConnectionUtil, eventBundleCreator)
+
+    /**
+     * Returns a new [FirebaseEventLogger] for the current application context.
+     *
+     * This [FirebaseEventLogger] implements the [PerformanceMetricsEventLogger] for facilitating
+     * performance metrics log reporting.
+     */
+    fun createPerformanceMetricEventLogger(): PerformanceMetricsEventLogger =
+      FirebaseEventLogger(firebaseAnalytics, networkConnectionUtil, eventBundleCreator)
```

Comment:
Is it okay for more than one of these loggers to exist simultaneously?

Reply:
I don't think there's any additional downside to having multiple loggers to exist simultaneously. However, if you feel that we shouldn't do this then we can possibly extend the existing EventLogger to implement performanceMetrics logging as well. WDYT?

Reply:
If there aren't concerns from a Firebase perspective, I think this approach is fine.